integration_test/tests/connection_tests.rs

```diff
@@ -21,4 +21,30 @@ mod integration {
         let conn_str = crate::common::generate_default_connection_str();
         env.connect_with_connection_string(&conn_str).unwrap();
     }
+
+    /**
+     * The following tests require a DSN called "ADF_Test" to be configured on the machine running the tests.
+     */
+
+    #[cfg(target_os = "windows")]
+    mod test_dsn {
+        use odbc::create_environment_v3;
+        #[test]
+        fn test_valid_dsn_connection() {
+            let env = create_environment_v3().unwrap();
+            let conn_str = "DSN=ADF_Test";
+            env.connect_with_connection_string(conn_str).unwrap();
```

Comment:
just to double check -- is this just confirming this doesn't panic? is there any expected value we should check against (i.e. if we stored the result, like in the following test)?

Reply:
Yeah, it will fail for numerous reasons (bad credentials or bad uri) if the info is wrong. I can link to a lot of build logs while I was figuring out how to massage the evergreen script to make this pass!